templates/ts/graph-connector/src/custom/getAclFromItem.ts

```diff
@@ -1,6 +1,11 @@
 import { Item } from "../models/Item";
 import { ExternalConnectors } from "@microsoft/microsoft-graph-types";
 
+// [Customization point]
+// If you need additional logic to set the ACL for the item, you can add it here
+// This function is used to set the ACL for each item
+// in the Graph API. The ACL is used to control access to the item.
+// For example, you can use a different ACL for different items, etc.
 export function getAclFromITem(item: Item): ExternalConnectors.Acl[] {
   return [
```

Comment:
I'd add a comment that here, we are making it public. This is important as if it's missed, people could bring sensitive data without securing it and it would be a big oversight.